src/themes/default/elements/icon.overrides

```diff
@@ -201,6 +202,19 @@ i.icon.other.gender.horizontal:before { content: "\f22b"; }
 i.icon.neuter:before { content: "\f22c"; }
 i.icon.genderless:before { content: "\f22d"; }
 
+/* Accessibility */
+i.icon.universal.access:before { content: "\f29a"; }
+i.icon.alternative.wheelchair:before { content: "\f29b"; }
+i.icon.blind:before { content: "\f29d"; }
+i.icon.audio.description:before { content: "\f29e"; }
```

Comment:
New icon: `audio description`, the assistive technology for additional description of audiovisual content. Are the capital letters A and D, with curved lines at the right side of the D, that represent waves of sound.